docs/source/api/plugin/landing-pages.md

```diff
@@ -106,6 +106,52 @@ By default, the landing page displays a footer that links to the documentation t
 <tr>
 <td>
 
+###### `document`
+
+`string`
+</td>
+<td>
+
+A GraphQL document (eg, query or mutation) to populate in the Studio Explorer's editor on load.
```

Comment:
```suggestion
A GraphQL document (eg, query or mutation) to populate in the Studio Sandbox Explorer's editor on load.
```

Reply:
Do we still want to add this? This API applies to both sandbox and explorer, right?

Reply:
This is the local dev docs, so they will only ever lead to sandbox afaict

Reply:
no strong opinion tho!